src/libraries/System.Drawing.Common/src/System/Drawing/BitmapSelector.cs

```diff
@@ -64,7 +64,7 @@ internal static string AppendSuffix(string filePath)
         /// </summary>
         public static string GetFileName(string originalPath)
         {
-            if (Suffix == string.Empty)
+            if (Suffix.Length == 0)
```

Comment:
```suggestion
            if (string.IsNullOrEmpty(Suffix))
```

Reply:
Just a small comment on this, where the Suffix is actually being set?
I'm trying to find usages of the setter but couldn't find.
And also, why not `Suffix?.Length` to keep the semantics as before?

Reply:
Comment says it's read by configuration. Using `IsNullOrEmpty` as following the logic through a `null` would just do pointless work to split the path into name + extension and then concatenate them. In concat `null` becomes `string.Empty` so you end up with the same FileName again.

https://github.com/dotnet/runtime/blob/ea6a008bad1737aa7df28973434504a6bca7c9c8/src/libraries/System.Drawing.Common/src/System/Drawing/BitmapSelector.cs#L52